Constitution.md

```diff
@@ -145,43 +141,65 @@ If there is more then 1 president, they exist as a single voting member on counc
 
 1. Oversee the planning the events for the Computer Science community.
 2. Coordinate a minimum of one event or academic initiative in each term, as per the discretion of the Council.
 3. Ensure there are an adequate number of social and academic events.
-4. Oversee and create posters and advertisements for all CSSA events.
-5. Ensure posters and advertisements for CSSA events are in an accessible format.
-6. Work in collaboration with other groups to promote and assist with collabroative events and initiatives.
-7. Ensure that the CSSA lounge is a safe and inclusive space.
-8. Lead the Events and Promotions Committee with up to 3 committee members.
+4. Work in collaboration with other groups to promote and assist with collaborative events and initiatives.
+5. Ensure that the CSSA lounge is a safe and inclusive space.
+6. Work with the Director of Promotions to create posters and advertisements for all CSSA events.
+7. Ensure posters and advertisements for CSSA events are in an accessible format.
+8. Work with the Director of Promotions to advertise group events on the CSSA's social media pages.
 9. Perform other duties incidental to this office within reasonable discretion as directed by the President and/or Council.
 
 ### Director of Advocacy
+If not filled by an elected executive in the CSSA elections, an elected executive of an UMSU recognized underrepresented community within the Department of Computer Science may be appointed as the Director of Advocacy for the CSSA, e.g., the Women in Computer Science Council.
 
 1. Act as a liason between CSSA and UMSU community representatives: 2SLGBTQIA+, Accessibility, Black, Indigenous, International, Mature, Racialized and Women.
-2. Attend Community Assemblies.
-3. Guides the council on advocacy for underrepresented communities.
-4. Elected by the student body in the CSSA elections.
-5. If not filled by an elected executive in the CSSA elections, An elected executive of an UMSU recognized underrepresented community within the Department of Computer Science may be appointed as the Director of Advocacy for the CSSA. Eg: The Women in Computer Science Council.
-6. Ensure that the CSSA lounge is a safe and inclusive space.
-7. Lead the Community Advocates commitee with up to and including committee members for each community recogized under UMSU.
+2. Attend UMSU Community Assemblies.
+3. Guide the council on advocacy for underrepresented communities.
+4. Ensure that the CSSA lounge is a safe and inclusive space.
+5. Oversee the election of Community Advocates and ensure that it is run competently, fairly, and transparently to CSSA's members.
+6. Work with, call meetings with, and delegate tasks to Community Advocates.
+7. Perform other duties incidental to this office within reasonable discretion as directed by the President and/or Council.
+
+### Director of Finances
+1. Act as a signing authority for CSSA.
+2. Providing a brief report on CSSA finances and expenses at each meeting.
```

Comment:
nit:

```suggestion
2. Provide a brief report on CSSA finances and expenses at each meeting.
```

but also are we sure we want to mandate this in the constitution? every single meeting?

Reply:
I think yes, even if the update is "no update this week" we should. A week is a bit time from finance perspective so accountability for them to staying up to date on bank account or any frauds etc.

Reply:
I'll say "regular updates" as to not overregulate, how about?